catalog/rest/catalog-rest-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -18,10 +18,10 @@
            http://cxf.apache.org/blueprint/jaxrs
            http://cxf.apache.org/schemas/blueprint/jaxrs.xsd">
 
-    <reference id="catalogService" interface="org.codice.ddf.rest.service.CatalogService"/>
+    <reference id="catalogService" interface="org.codice.ddf.rest.service.CatalogService" availability="optional"/>
```

Comment:
❓ Is this now optional? Can this module function without having a reference of this?

Reply:
Fixed